### ISSUE TITLE
Apistar requires ujson which is not listed in setup.py

### ISSUE THREAD

**Issue by sylwekb** [tomchristie/apistar#38]
command `apistar new --template minimal` returns:
```bash
~/devment/apistar » apistar new --template minimal                                                                                                                                                                 [apistar]sylwesterbrzeczkowski@Sylwesters-MacBook-Pro
Traceback (most recent call last):
  File "/Users/sylwesterbrzeczkowski/.virtualenvs/apistar/bin/apistar", line 7, in <module>
    from apistar.main import main
  File "/Users/sylwesterbrzeczkowski/.virtualenvs/apistar/lib/python3.6/site-packages/apistar/__init__.py", line 8, in <module>
    from apistar.app import App
  File "/Users/sylwesterbrzeczkowski/.virtualenvs/apistar/lib/python3.6/site-packages/apistar/app.py", line 5, in <module>
    from apistar import commands, routing
  File "/Users/sylwesterbrzeczkowski/.virtualenvs/apistar/lib/python3.6/site-packages/apistar/routing.py", line 9, in <module>
    from apistar import app, http, pipelines, schema, wsgi
  File "/Users/sylwesterbrzeczkowski/.virtualenvs/apistar/lib/python3.6/site-packages/apistar/http.py", line 10, in <module>
    import ujson as json
ModuleNotFoundError: No module named 'ujson'
```

**Comment by tomchristie** [comment]
Closed via #39. Release to follow. Thank you!

**Comment by tomchristie** [comment]
Now available on PyPI.
